models/block.coffee

```diff
@@ -1,17 +1,10 @@
-#
-# Model for a single block
-#
-
 Base = require "./base.coffee"
 sd = require("sharify").data
 _ = require 'underscore'
 _s = require 'underscore.string'
 moment = require 'moment'
-parseDomain = require 'parse-domain'
```

Comment:
Sort of funny, this was probably the biggest dependency in the entire bundle and it was never used.